src/backend/src/utils/auth.utils.ts

```diff
@@ -36,9 +37,14 @@ export const requireJwtProd = (req: Request, res: Response, next: any) => {
 
     if (!token) return res.status(401).json({ message: 'Authentication Failed: Cookie not found!' });
 
-    jwt.verify(token, TOKEN_SECRET, (err: any, decoded: any) => {
+    //uses the VerifyCallback type in the index.d.ts file for err and decoded types.
+    //Only jwtPayload of the decoded types has a userID, so other types throw an error
```

Comment:
Sorry, I meant put these comments on the PR for our reference, these are probably a bit too detailed for comments in the code